src/alire/alire-os_lib-subprocess.ads

```diff
@@ -19,6 +19,15 @@ package Alire.OS_Lib.Subprocess is
       Understands_Verbose : Boolean := False;
       Err_To_Out          : Boolean := False) return Utils.String_Vector;
    --  Either suceeds or raises Checked_Error with the code and output as info.
-   --  Output is captured and returned.
+   --  Output is captured and returned on success.
+
+   procedure Checked_Spawn_And_Capture
```

Comment:
I am not sure why you are making a procedure version of this function.

Reply:
This is to be able to log the partial output of the spawned command in case of failure.

Reply:
ok I understand